packages/themes/scss/_tokens.scss

```diff
@@ -1,327 +1,8 @@
-// Code generated by @carbon/themes. DO NOT EDIT.
 //
```

Comment:
Is it possible to rename the `_tokens.scss` files here to `_variables.scss` without a breaking change? The direction that we want to go: each package has `index.scss` and `package-name.scss` entrypoints that you choose based on your build. When using the package's Sass, imports are deterministic so if you are looking to include a Sass variable without side-effects, you `@import './path/to/variables.scss';` and you don't have to look up if the file name is actually called variables, or tokens, global-tokens, etc.

Reply:
I'm definitely torn on the file naming scheme by syntax instead of feature, maybe this could be a broaden change across the project?

Reply:
It needs to be a broad change across all packages (especially components). If you're building one giant Sass file for your app, it doesn't matter because everything has previously been imported and is available. But with that, a poor development experience with slow re-builds of everything Sass.

When styling in a more modular fashion (e.g. CSS modules), each file can be its own Sass build. Great developer experience, but many more imports. "I need to use the breakpoint **mixin** for this, spacing **variables** for that, a theme **variables** here, theme **mixin** there"... deterministic imports really help.

As this PR is only about the themes package, we can defer a file naming scheme until later. My suggestion is based on what I see in the industry and that if we do it now, we don't have to do it later. It doesn't have to happen now if it requires a separate discussion. Your call.